README.md

```diff
@@ -174,10 +187,25 @@ Run the following script to create a new entry that you will update with the sam
 ./scripts/update-changelog securedrop-foobar
 ```
 
-Finally, build the package by pointing to the tarball and package version:
+First verify the tarball you are about to package into a deb:
+
+```
+gpg --verify <package>.tar.gz.asc <package>.tar.gz
```

Comment:
If you elect to add shell expansion change above, should be added here too.

Reply:
I think we can leave the shell expansion trick out of the docs since it's clearer without it